sonic-net/sonic-buildimage
# Streaming structured events implementation

#### Why I did it
Implementation of HLD [PR #954](https://github.com/sonic-net/SONiC/pull/954)
Depends on swsscommon [PR #649](https://github.com/Azure/sonic-swss-common/pull/649) and corresponding submodule update.


#### How I did it

#### How to verify it
With this PR in, you flap BGP and use events_tool to see the published events.
With telemetry [PR #111](https://github.com/Azure/sonic-telemetry/pull/111) in and corresponding submodule update done in buildimage, one could run gnmi_cli to capture BGP flap events.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
